test/integration/022_bigquery_test/copy-models/schema.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+sources:
+  - name: test_copy_several_tables
+    schema: "{{ target.schema) }}"
```

Comment:
```suggestion
    schema: "{{ target.schema }}"
```

Reply:
After I fix this locally, and run `test_bigquery_copy_models.py`, I see `copy_as_several_tables` failing with:
```
Database Error in model copy_as_several_tables (copy-models/copy_as_several_tables.sql)
  Table dbt-test-env:test16303148312675888629_bigquery_test_022.original appears multiple times in source tables.
```
Do you have a sense of what that error indicates?

Reply:
Fixed